Mathlib/RingTheory/MvPolynomial/Symmetric.lean

```diff
@@ -210,10 +220,18 @@ theorem esymm_eq_sum_monomial (n : ℕ) :
 #align mv_polynomial.esymm_eq_sum_monomial MvPolynomial.esymm_eq_sum_monomial
 
 @[simp]
-theorem esymm_zero : esymm σ R 0 = 1 := by
-  simp only [esymm, powersetCard_zero, sum_singleton, prod_empty]
+theorem esymm_zero : esymm σ R 0 = 1 := by simp [esymm]
 #align mv_polynomial.esymm_zero MvPolynomial.esymm_zero
 
+@[simp]
+theorem esymm_one : esymm σ R 1 = ∑ i, X i := by simp [esymm, powersetCard_one]
+
+theorem esymmPart_zero : esymmPart σ R (.indiscrete 0) = 1 := by simp [esymmPart]
+
+@[simp]
+theorem esymmPart_onePart (n : ℕ) : esymmPart σ R (.indiscrete n) = esymm σ R n := by
```

Comment:
I don't really understand this name. What is the `onePart` referring to.

Reply:
I missed it after renaming "onePart" to "indiscrete". Fixed that.